src/app/components/Table/Table.tsx

```diff
@@ -51,9 +51,14 @@ export const Table = <RowDataType extends Record<never, unknown>>({
 		return <tr />;
 	};
 
-	const renderHeaderGroup = (headerGroup: HeaderGroup<RowDataType>) => (
-		<tr {...headerGroup.getHeaderGroupProps()}>{headerGroup.headers.map(renderColumn)}</tr>
-	);
+	const renderHeaderGroup = (headerGroup: HeaderGroup<RowDataType>) => {
+		const { key, ...headerGroupProps } = headerGroup.getHeaderGroupProps();
+		return (
+			<tr key={key} {...headerGroupProps}>
+				{headerGroup.headers.map(renderColumn)}
+			</tr>
+		);
```

Comment:
This is to fix warning:

![image](https://github.com/user-attachments/assets/ff967393-38ea-4b4d-8709-b69d01d53d6c)